shared/js/background/chrome-events.es6.js

```diff
@@ -232,7 +249,7 @@ chrome.webNavigation.onCommitted.addListener(details => {
             }
             chrome.tabs.executeScript(details.tabId, variableScript)
             const scriptDetails = {
-                'file': '/data/fingerprint-protection.js',
+                'file': '/public/js/injected-content-scripts/fingerprint-protection.js',
```

Comment:
It may make sense to inject this file statically always and initialize it like this:
```
initFingerprintProtection(JSON.stringify({
  agent: agentSpoofer.getAgent(),
  sessionKey,
  referrer: tab.referrer
}));
```